app/test_main.py

```diff
@@ -1,3 +1,58 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestAgeForAnimal:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 27, [2, 2]),
+            (28, 28, [3, 2]),
+            (100, 100, [21, 17])
+        ]
+    )
+    def test_get_age_correctly(self, cat_age: int, dog_age: int, expected: list) -> None:
+        """Test if the function correctly converts cat and dog ages to human years."""
+        assert get_human_age(cat_age, dog_age) == expected
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected",
+        [
+            (-1, 5, [0, 0]),
+            (5, -1, [0, 0]),
```

Comment:
The expected result for negative ages is set to `[0, 0]`. This might not be the correct behavior for the `get_human_age` function unless it is explicitly designed to return `[0, 0]` for negative inputs. Consider revisiting the logic of `get_human_age` to ensure it handles negative ages appropriately.